docker/packager/deb/build.sh

```diff
@@ -10,4 +10,15 @@ mv *.changes /output
 mv *.buildinfo /output
 mv /*.rpm /output ||: # if exists
 mv /*.tgz /output ||: # if exists
+
+if [ -n "$BINARY_OUTPUT" ] && { [ "$BINARY_OUTPUT" = "programs" ] || [ "$BINARY_OUTPUT" = "tests" ] ;}
+then
+  echo Place $BINARY_OUTPUT to output
+  mkdir /output/binary ||: # if exists
+  mv /build/obj-x86_64-linux-gnu/programs/clickhouse* /output/binary
```

Comment:
Why do we want to publish the copy binaries outside of `.deb` package? It will be uploaded to s3 and require twice a space. Also, these binaries possibly stripped inside the `release` script and not very useful.

Reply:
Otherwise we need to build unit tests with each sanitizer in separate build task. I wanted to run build only once for non release build types (debug/*san). 

I'll check if strip is disabled in such builds

Reply:
ok, let's check

Reply:
Added nosplit for builds with tests.